app/views/health/patient/_remaining_to_engage.haml

```diff
@@ -41,9 +43,10 @@
           %span.icon-checkmark.mr-1.o-color--gray-l
           Release of Information form
   %li
-    - # SSMs are valid for one year (if they occur after the enrollment date)
+    - # SSMs are valid if the signature is within 1 year prior to enrollment start date, and are valid for
+    - # one year from the signature date
     - ssm = patient.self_sufficiency_matrix_forms.active.recent.during_current_enrollment.first
```

Comment:
Same here, `ssm = patient.self_sufficiency_matrix_forms.active.recent.during_current_enrollment.first` seems like it may never hit the 1 year check.